tests/Unit/Domain/CoordinateMaps/TimeDependent/Test_RotScaleTrans.cpp

```diff
@@ -214,427 +250,288 @@ void test_RotScaleTrans() {
+      test_jacobian(trans_map_inner, point_to_check, t, f_of_t_list);
+      test_inv_jacobian(trans_map_inner, point_to_check, t, f_of_t_list);
+      test_jacobian(trans_map_transition, point_to_check, t, f_of_t_list);
+      test_inv_jacobian(trans_map_transition, point_to_check, t, f_of_t_list);
+      test_jacobian(trans_map_outer, point_to_check, t, f_of_t_list);
+      test_inv_jacobian(trans_map_outer, point_to_check, t, f_of_t_list);
+      test_jacobian(rot_scale_map_inner, point_to_check, t, f_of_t_list);
+      test_inv_jacobian(rot_scale_map_inner, point_to_check, t, f_of_t_list);
+      test_jacobian(rot_scale_map_transition, point_to_check, t, f_of_t_list);
+      test_inv_jacobian(rot_scale_map_transition, point_to_check, t,
+                        f_of_t_list);
+      test_jacobian(rot_scale_map_outer, point_to_check, t, f_of_t_list);
+      test_inv_jacobian(rot_scale_map_outer, point_to_check, t, f_of_t_list);
+      test_jacobian(rot_trans_map_inner, point_to_check, t, f_of_t_list);
+      test_inv_jacobian(rot_trans_map_inner, point_to_check, t, f_of_t_list);
+      test_jacobian(rot_trans_map_transition, point_to_check, t, f_of_t_list);
+      test_inv_jacobian(rot_trans_map_transition, point_to_check, t,
                         f_of_t_list);
-      test_jacobian(rot_scale_trans_map_rigid, far_point_xi, t, f_of_t_list);
-      test_inv_jacobian(rot_scale_trans_map_rigid, far_point_xi, t,
+      test_jacobian(rot_trans_map_outer, point_to_check, t, f_of_t_list);
+      test_inv_jacobian(rot_trans_map_outer, point_to_check, t, f_of_t_list);
+      test_jacobian(scale_trans_map_inner, point_to_check, t, f_of_t_list);
+      test_inv_jacobian(scale_trans_map_inner, point_to_check, t, f_of_t_list);
+      test_jacobian(scale_trans_map_transition, point_to_check, t, f_of_t_list);
+      test_inv_jacobian(scale_trans_map_transition, point_to_check, t,
                         f_of_t_list);
-      test_jacobian(rot_scale_trans_map_non_rigid, far_point_xi, t,
+      test_jacobian(scale_trans_map_outer, point_to_check, t, f_of_t_list);
+      test_inv_jacobian(scale_trans_map_outer, point_to_check, t, f_of_t_list);
+      test_jacobian(rot_scale_trans_map_inner, point_to_check, t, f_of_t_list);
+      test_inv_jacobian(rot_scale_trans_map_inner, point_to_check, t,
+                        f_of_t_list);
+      test_jacobian(rot_scale_trans_map_transition, point_to_check, t,
                     f_of_t_list);
-      test_inv_jacobian(rot_scale_trans_map_non_rigid, far_point_xi, t,
+      test_inv_jacobian(rot_scale_trans_map_transition, point_to_check, t,
+                        f_of_t_list);
+      test_jacobian(rot_scale_trans_map_outer, point_to_check, t, f_of_t_list);
+      test_inv_jacobian(rot_scale_trans_map_outer, point_to_check, t,
                         f_of_t_list);
+    };
+
+    if (radius <= inner_radius * .99) {
+      check_inner_maps(point_xi);
+    } else if (radius >= inner_radius * 1.01) {
```

Comment:
Why the buffer? I thought that was only needed for the jacobian tests. Also, isn't `radius >= inner_radius * 1.01` excluding the specific part of the map we want to test the most? Where it goes bad because of roundoff?